data/global/excel/runes.txt

```diff
@@ -140,42 +140,43 @@ Runeword117	Rain	1				circ									BerChamBer	r30	r32	r30				levelreq		22	22	all
 Runeword157	Void	1				barb									SolAmn	r12	r11					bar		1	1	dmg-max		10	15	hp		50	50	balance2		25	25	move2		25	25	lifesteal		3	6	manasteal		3	6	0
 Runeword158	War	1				helm									IoHel	r16	r15					ac%		100	140	dmg%		35	50	swing3		30	30	red-dmg%		20	25	ignore-ac		1	1	noheal		1	1	balance2		25	25	0
-Runeword159	Water	1				spea									KoShaelHel	r18	r13	r15				dmg-cold	150	33	122	regen-mana		35	35	regen		3	5	dmg%		175	175	balance2		25	25	levelup-skill	battle orders	100	25	res-fire		50	50	0
+Runeword159	Water	1				spea									KoShaelHel	r18	r13	r15				dmg-cold	150	33	122	regen-mana		35	35	regen		3	5	dmg%		175	175	balance2		25	25	levelup-skill	Battle Orders	100	25	res-fire		50	50	0
 Runeword161	Whisper	1				weap									ShaelShaelShael	r13	r13	r13				dmg-norm		25	75	dmg%		100	100	regen		6	6	reduce-ac		15	15	ease		-25	-25									0
 Runeword47	Widowmaker	1				swor	axe								EthTirLoMalRal	r05	r03	r28	r23	r08		hit-skill	278	20	15	swing2		30	40	dmg		340	400	ignore-ac		1	1	dmg-dem/lvl	15			pierce-pois		20	25	heal-kill		11	11	0
 Runeword163	Wind	1			110	mele									SurEl	r29	r01					dmg%		120	160	swing2		40	40	move2		20	20	reduce-ac		50	50	hit-skill	Tornado	10	9	charged	240	127	13	balance2		15	15	0
-Runeword164	Wings of Hope	1				scep									ChamCham	r32	r32					dmg-min		60	100	dmg-max		150	200	oskill	teleport	8	8	aura	125	3	3	hit-skill	84	25	35	rip		1	1	all-stats		20	20	0
+Runeword164	Wings  of  Hope	1				scep									ChamCham	r32	r32					dmg-min		60	100	dmg-max		150	200	oskill	Teleport	8	8	aura	125	3	3	hit-skill	84	25	35	rip		1	1	all-stats		20	20	0
 Runeword99	Winter	1				swor	mace								LemKoElEld	r20	r18	r01	r02			hit-skill	64	15	13	hit-skill	45	18	20	dmg-demon		220	350	dmg-undead		280	300	dmg-cold	75	100	220	pierce-cold		24	24	nofreeze		1	1	0
-Runeword165	Wisdom	1			D2R Ladder 1	helm									PulIthEld	r21	r06	r02				pierce		33	33	manasteal		4	8	att%		15	25	mana-kill		5	5	nofreeze		1	1	enr		10	10					0
+Runeword165	Wisdom	1			 D2R  Ladder  1 	helm									PulIthEld	r21	r06	r02				pierce		33	33	manasteal		4	8	att%		15	25	mana-kill		5	5	nofreeze		1	1	enr		10	10					0
 Runeword166	Woe	1				tors									ChamTirShael	r32	r03	r13				levelreq	7	7		gethit-skill	53	25	25	gethit-skill	47	25	25	gethit-skill	59	25	25	allskills		2	2	all-stats		25	25	res-pois-len		80	80	0
 Runeword167	Wonder	1				mele									VexAmnShaelAmnVex	r26	r11	r13	r11	r26		dmg%		275	275	balance2		30	30	str		20	20	dex		20	20	gold%/time	1	1	200	mag%/time	3	1	100	hp/lvl	12			0
-Runeword168	Wrath	1			Previously Ladder Only	miss									PulLumBerMal	r21	r17	r30	r23			dmg-demon		300	300	dmg-undead		250	300	dmg-ltng		41	240	dmg-mag		85	120	hit-skill	Life Tap	5	10	hit-skill	Decrepify	30	1	nofreeze		1	1	0
+Runeword168	Wrath	1			 Previously  Ladder  Only 	miss									PulLumBerMal	r21	r17	r30	r23			dmg-demon		300	300	dmg-undead		250	300	dmg-ltng		41	240	dmg-mag		85	120	hit-skill	Life Tap	5	10	hit-skill	Decrepify	30	1	nofreeze		1	1	0
```

Comment:
It still looks like it's adding a lot of spaces from the ones I removed previously.